### PR TITLE
CLI commands to add credentials in the workspace keyvault 

### PR DESCRIPTION
**Pull request by garg-amit**
## Purpose

* CLI commands to add credentials in the workspace key vault 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: docs/real-world-examples/ccfraud.md [hunk 1]
```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
+
+1. Get the signed-in user's object ID.
```
Comment (jfomhover) [inline docs/real-world-examples/ccfraud.md:36]: Can you add a context here?

Ex:

"In this section, we'll create an access policy for yourself, by using your AAD identifier (object id)"

File: docs/real-world-examples/ccfraud.md [hunk 2]
```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
+
+1. Get the signed-in user's object ID. 
+```bash
+az ad signed-in-user show | jq ".id"
+```
+2. Grant permissions to *list, set & delete* secrets by setting a keyvault policy.
```
Comment (jfomhover) [inline docs/real-world-examples/ccfraud.md:40]: rewrite:

Create a new keyvault policy for yourself, and grant permissions to *list, set & delete* secrets.

File: docs/real-world-examples/ccfraud.md [hunk 3]
```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
...
+1. Get the signed-in user's object ID. 
+```bash
+az ad signed-in-user show | jq ".id"
+```
+2. Grant permissions to *list, set & delete* secrets by setting a keyvault policy. 
+```bash
+az keyvault set-policy -n <key-vault-name> --secret-permissions list set delete --object-id <object-id>
+```
+> Note: The AML workspace you created with the aforementioned script contains the name of the key vault. Default is `kv-fldemo`.
+3. Create a secret to store the `kaggleusername`.
```
Comment (jfomhover) [inline docs/real-world-examples/ccfraud.md:45]: With your newly created permissions, you can now create...

File: docs/real-world-examples/ccfraud.md [hunk 4]
```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
```
Comment (jfomhover) [inline docs/real-world-examples/ccfraud.md:34]: Can you change title "Option 1 : using Azure CLI" and same for Option 2 ? so people understand those are 2 different paths

File: docs/real-world-examples/ccfraud.md [hunk 5]
```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
...
+```bash
+az keyvault secret set --name kagglekey --vault-name <key-vault-name> --value <kaggle-api-token>
+```
+> Make sure to provide the *Kaggle API Token*.
+
+### Using Azure UI
+
 1. In your workspace resource group (provisioned in the previous step), open "Access Policies" tab in the newly keyvault.
 
 2. Select "Select all" right under "Secret Management Operations" and press "Next".
```
Comment (jfomhover) [inline docs/real-world-examples/ccfraud.md:60]: Can you modify this to include just list, set and delete?

**Review by jfomhover — CHANGES_REQUESTED** [review]
Here's a couple comments to enhance the tutorial, thank youfor putting it up!

(consider my comments on ccfraud to map to pneumonia as well)

**Review by jfomhover — APPROVED** [review]

File: docs/real-world-examples/ccfraud.md [hunk 6]
```diff
@@ -31,9 +31,33 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Option 1: using Azure CLI
+
+1. In this section, we'll create an access policy for yourself, by using your AAD identifier (object id). 
+```bash
+az ad signed-in-user show | jq ".id"
+```
+2. Create a new keyvault policy for yourself, and grant permissions to list, set & delete secrets.
```
Comment (thomasp-ms) [inline docs/real-world-examples/ccfraud.md:40]: here (and maybe elsewhere in the text?): key vault (2 words), not keyvault

File: docs/real-world-examples/ccfraud.md [hunk 7]
```diff
@@ -31,9 +31,33 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Option 1: using Azure CLI
+
+1. In this section, we'll create an access policy for yourself, by using your AAD identifier (object id).
```
Comment (thomasp-ms) [inline docs/real-world-examples/ccfraud.md:36]: This sentence is a bit confusing. Maybe something like: "1. Let's first obtain your AAD identifier (object id) by running the following command. We'll use it in the next step."

**Review by thomasp-ms — APPROVED** [review]
2 minor comments, up to you to decide whether you want to address or not. This will probably create some merge conflicts for the bug bash feedback PR I have out. Let's merge yours first since it was ready first, then I'll fix mine.

**Pull request merged**